doc/news/changes.h

```diff
@@ -170,6 +170,17 @@ inconvenience this causes.
   (Timo Heister, 2015/09/30)
   </li>
 
+  <li> Improved: The interface to all deal.II type solvers and preconditioners have
+  been updated such that they function as expected with the LinearOperator class and
+  its associated functions (i.e. linear_operator, transpose_operator and
+  inverse_operator). These preconditioners can now be wrapped as a LinearOperator,
```

Comment:
I don't think doxygen will properly link to these names. I think they're in a namespace, right? If so, please prefix with the namespace and add `()` to the end to make it clear that these are functions.